dbms/src/Storages/StorageMergeTree.cpp

```diff
@@ -99,7 +99,7 @@ void StorageMergeTree::startup()
     /// NOTE background task will also do the above cleanups periodically.
     time_after_previous_cleanup.restart();
     merging_mutating_task_handle = global_context.getBackgroundPool().addTask([this] { return mergeMutateTask(); });
-    moving_task_handle = global_context.getBackgroundPool().addTask([this] { return movePartsTask(); });
+    moving_task_handle = global_context.getMovePool().addTask([this] { return movePartsTask(); });
```

Comment:
Not sure if we should start 8 extra threads even for cases when no multidisk config is in place.

Reply:
+